git_machete/git_operations.py

```diff
@@ -223,8 +223,8 @@ def _run_git(self, git_cmd: str, *args: str, flush_caches: bool, allow_non_zero:
         return exit_code
 
     def _popen_git(self, git_cmd: str, *args: str,
-                   allow_non_zero: bool = False, env: Optional[Dict[str, str]] = None) -> CommandResult:
-        exit_code, stdout, stderr = utils.popen_cmd("git", git_cmd, *args, env=env)
+                   allow_non_zero: bool = False, env: Optional[Dict[str, str]] = None, input: Optional[str] = None) -> CommandResult:
```

Comment:
```suggestion
                   allow_non_zero: bool = False, env: Optional[Dict[str, str]] = None, stdin: Optional[str] = None) -> CommandResult:
```

For consistency with `stdout, stderr` below